internal/rollbar/rollbar.go

```diff
@@ -23,9 +23,23 @@ type config interface {
 	Closed() bool
 }
 
-var currentCfg config
+type doNotReport []string
 
-var reportingDisabled bool
+func (d *doNotReport) Add(msg string) {
```

Comment:
If you accidentally pass `locale.T("does_not_exist")`, does it return an empty string? If so, would that end up blocking all rollbar reports? If so, we might want to ignore empty strings here and possibly report them to rollbar so we can fix.

Reply:
Thank you for catching this! It should be addressed now and I've added some tests to verify.